/**:
  ros__parameters:
    base_frame: base_link
    use_imu: true
    use_3d_distortion_correction: false
    update_azimuth_and_distance: false
    has_static_tf_only: true
